app/code/Magento/PageCache/Test/Mftf/Test/CachedInputFormKeyValueUpdatedTest.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CachedInputFormKeyValueUpdatedTest">
+        <annotations>
+            <features value="PageCache"/>
+            <stories value="FormKey"/>
+            <title value="Form Key value should be updated by js script"/>
+            <description value="Form Key value should be updated by js script"/>
+            <testCaseId value="MC-39300"/>
+            <useCaseId value="MC-30171"/>
+            <severity value="AVERAGE"/>
+            <group value="full_page"/>
+        </annotations>
+        <before>
+            <!-- Create Data -->
+            <createData entity="_defaultCategory" stepKey="createCategory"/>
```

Comment:
I see you actually don't need this category. Use SimpleProduct2 dataset to create product. This dataset dont need category parameter

Reply:
removed